packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx

```diff
@@ -142,7 +142,7 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
   private _getAddNewElement(): JSX.Element {
     let { personaSize } = this.props;
     return <button { ...getNativeProps(this.props.addButtonProps, buttonProperties) }
-      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-addButton', 'ms-Facepile-itemButton') }>
+      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-addButton', 'ms-Facepile-itemButton', styles.addButton, styles.itemButton) }>
```

Comment:
Hmmm....well i think this is going to cause the bug I saw once combined with persona.

I.e. We can pass classes down to child components, we need to pass data down that triggers the addition of styles.